src/activities/Elsa.Activities.Startup/Activities/Startup.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Elsa.Attributes;
+using Elsa.Expressions;
+using Elsa.Extensions;
+using Elsa.Results;
+using Elsa.Services;
+using Elsa.Services.Models;
+using NodaTime;
+
+namespace Elsa.Activities.Startup.Activities
+{
+    [ActivityDefinition(
+        Category = "Startup",
+        Description = "Triggers at startup."
+    )]
+    public class Startup : Activity
+    {
+
+        public Startup()
+        {
+        }
+
+
+        protected override ActivityExecutionResult OnExecute(WorkflowExecutionContext context)
+        {
+            return Halt();
```

Comment:
I'm not sure I understand how this works. It looks like that the activity will halt, but I don't see at which point the activity will be resumed. The `StartupHostedService` will trigger workflows starting with the `Startup` activity, which will cause the `OnExecute` method to be invoked. But then the workflow becomes suspended (the new term for "halted" in Eldsa 2.0).

Unless I'm missing something?

Reply:
Dunno, copied from timer activities. It works :-D
Shall I do onexecute instead?

Reply:
I see. So here's the thing: the Timer activities work because they are repeatedly being triggered. Internally, thee activities compare the current time with the first time they got triggered. If the difference exceeds a given threshold, the activity executes and continues the workflow.

The Startup background task on the other hand only triggers the Startup activity once. Since the Startup activity returns `HaltResult`, it will cause the workflow to be suspended. Only when you kill the ost and restart it, will the workflow continue. 

At least, this is the behavior I'd expect to see by looking at the code.

To summarize: please do return `Done()` from `OnExecute` :)

Reply:
Fixed :-)